modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -692,13 +807,13 @@ PERF_TEST_P_(CmpWithScalarPerfTest, TestPerformance)
 
 PERF_TEST_P_(BitwisePerfTest, TestPerformance)
 {
-    MatType   type           = -1;
-    bitwiseOp opType         = AND;
-    bool      testWithScalar = false;
+    compare_f cmpF;
+    bitwiseOp opType;
+    bool testWithScalar;
     cv::Size sz;
+    MatType type;
```

Comment:
Please add initialization with default values. See example in the `MulPerfTest` case (above). Also please **apply this comment to all test cases in this file**.

Reply:
Applied, thanks